content/partials/types/_location_pushes.textile

```diff
@@ -0,0 +1,7 @@
+Starting with iOS 15, Apple supports a power-efficient way to request location through the location push service extension.
+
+To receive location push notifications:
+
+# Request a location-specific device token by calling @CLLocationManager.startMonitoringLocationPushes(completion:)@ within the @ARTPushRegistererDelegate.didActivateAblyPush(:)@ delegate method. This delegate method is the callback for @ARTRealtime.push.activate()@.
```

Comment:
When rendered in the browser, this looks a little hard to follow. Would it be better to have the numbered point explaining and then below and indented, a code block with this in action?

Reply:
The table format seems to work much better:

<img width="635" height="383" alt="image" src="https://github.com/user-attachments/assets/3fe901b6-7fa3-40ed-ba69-d8ce988efa7b" />

Reply:
e18a96b854aed8d55136c9f25c65a8f3298911de